sockeye/config.py

```diff
@@ -133,3 +136,15 @@ def copy(self, **kwargs):
         for name, value in kwargs.items():
             object.__setattr__(copy_obj, name, value)
         return copy_obj
+
+    def disable_dropout(self):
+        """
+        Sets the value of all attributes that contain 'dropout' to 0.0.
+        """
+        for attr, val in self.__dict__.items():
+            if 'dropout' in attr:
+                logger.debug("Setting %s to 0.0", attr)
+                setattr(self, attr, 0.0)
+
+            elif isinstance(val, Config):
```

Comment:
I would change the order of the checks. If I see it correctly in the current form, if a subconfig has an attribute called 'dropout' it won't recurse into it.

Reply:
Good point about the edge case of a subconfig named 'dropout' :)
Will revise.